FileManipulator.Test/GlobalJsonManipulatorTest.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Messerli.FileManipulatorAbstractions;
+using Messerli.FileSystem;
+using Messerli.Test.Utility;
+using Xunit;
+
+namespace Messerli.FileManipulator.Test
+{
+    public sealed class GlobalJsonManipulatorTest
+    {
+        private const string FileName = "global.json";
+        private const string MsBuildSdkNugetPackageId = "Microsoft.Build.NoTargets";
+        private const string MsBuildSdkVersion = "1.0.0";
+        private static readonly string NewLine = Environment.NewLine;
+
+        private static readonly GlobalJsonModification AddMsBuildSdkModification =
+            new GlobalJsonModificationBuilder()
+                .AddMsBuildSdk(new MsBuildSdk(MsBuildSdkNugetPackageId, MsBuildSdkVersion))
+                .Build();
+
+        [Theory]
+        [MemberData(nameof(GetModificationData))]
+        public async Task AddsMsBuildSdksToConfig(string expectedConfig, string? existingConfig, GlobalJsonModification modification)
+        {
+            using var testEnvironment = new TestEnvironmentProvider();
+            var filePath = Path.Combine(testEnvironment.RootDirectory, FileName);
+
+            if (existingConfig is { })
```

Comment:
Split this into two theories

Reply:
https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/28a5dc06d2261711fb152a4e7e14c7d027c46486